src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -99,6 +100,16 @@ function VAFacilityPageV2({
     [isLoading],
   );
 
+  const previouslyShowingModal = usePrevious(showEligibilityModal);
+  useEffect(
+    () => {
+      if (!showEligibilityModal && previouslyShowingModal) {
+        scrollAndFocus('.usa-button-primary');
```

Comment:
Focus on the continue button when the user closes the eligibility modal, since that's what they clicked that triggered the modal